packages/ckeditor5-media-embed/src/mediaembedui.ts

```diff
@@ -40,74 +39,95 @@ export default class MediaEmbedUI extends Plugin {
 	 */
 	public init(): void {
 		const editor = this.editor;
-		const command: MediaEmbedCommand = editor.commands.get( 'mediaEmbed' )!;
 
-		editor.ui.componentFactory.add( 'mediaEmbed', locale => {
-			const dropdown = createDropdown( locale );
+		editor.ui.componentFactory.add( 'mediaEmbed', () => {
+			const button = this._createDialogButton( ButtonView );
 
-			this._setUpDropdown( dropdown, command );
+			button.set( {
+				tooltip: true
+			} );
```

Comment:
```suggestion
			button.tooltip = true;
```

When we set only one value we rather use the shorthand form.